ale/drivers/base.py

```diff
@@ -103,15 +409,28 @@ def line_scan_rate(self):
     @property
     def number_of_ephemerides(self):
         #TODO: Not make this hardcoded
-        return 909
+        return 909 #len(self._sensor_position)
 
     @property
     def number_of_quaternions(self):
         #TODO: Not make this hardcoded
-        return 909
+        return 909 #
+        len(self._quaternions)
```

Comment:
Same as number_of_ephemerides. Why is the real value commented out?

Reply:
same reason